charms/kfp-api/src/charm.py

```diff
@@ -40,18 +42,19 @@ def __init__(self, *args):
 
         self.prometheus_provider = MetricsEndpointProvider(
             charm=self,
-            relation_name="monitoring",
+            relation_name="metrics-endpoint",
             jobs=[
                 {
-                    "job_name": "ml_pipeline",
-                    "scrape_interval": self.config["metrics-scrape-interval"],
-                    "metrics_path": self.config["metrics-api"],
+                    "metrics_path": METRICS_PATH,
```

Comment:
Doesn't hurt to have this, but also this is the default anyway.  This can be removed

Reply:
Can it be kept here since other charms also have this specified, even though it's default?